domains/38middle.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Documentation for is-a-dev",
+  "owner": {
+    "username": "SoftOSDev",
+    "email": "3093634343@qq.com"
+  },
+  "record": {
+    "CNAME": "softosdev.github.io",
+    "A": [
+      "185.199.111.153",
+      "185.199.110.153",
+      "185.199.109.153",
+      "185.199.108.153"
+    ]
```

Comment:
You cannot use CNAME with A records (just pick one of them and remove the other one)